material-overrides/assets/stylesheets/kluster.css

```diff
@@ -1361,6 +1363,11 @@ span.badge.integration {
 
 .md-typeset .grid.cards > ul > li > hr {
   border-bottom: 0.05rem solid var(--primary-green-300);
+  margin: 0.5em 0;
+}
+
+.md-typeset .grid.cards > ul > li > :last-child {
+    margin-top: auto;
```

Comment:
I don't think we always want to make it the last child. We want to make sure the last child has an `a` tag. So something like this would be better:

```
.md-typeset .grid.cards > ul > li > :last-child:has(> a)
```

Reply:
Also please make sure the formatting of this is all good, looks like theres 4 spaces for the indentation here but usually we use 2